yaml/contours.yaml

```diff
@@ -0,0 +1,15 @@
+global:
+  - &output_properties
+    source: {col: source}
+    min_zoom: {col: min_zoom}
+    ele: {col: ele}
```

Comment:
We generally spell out property names, so `elevation` here instead of `ele`.

Reply:
1. Generally we want to assign an index contour that's every say 4th or 5th value... so **0**, 20, 40, 60, 80, **100** where 0 and 100 would be index contours. This allows the index contours to be "bold" (or thicker stroke) to establish better visual hierarchy on the map display, and the text can also be styled bold. 

So we'll probably need a new transform to assign `kind_detail` when it's an index contour.

Reply:
Updated according to your first comment.

Reply:
I am not sure how to add this new transfor to assign `kind_detail`. Is this also needed? Because in current styles I am doing this in function in scene styling files.